code/datums/martial.dm

```diff
@@ -0,0 +1,293 @@
+								"<span class='userdanger'>[A] has hit [D] with Plasma Punch!</span>")
+	playsound(D.loc, 'sound/weapons/punch1.ogg', 50, 1, -1)
+	var/atom/throw_target = get_edge_target_turf(D, get_dir(D, get_step_away(D, A)))
+	D.throw_at(throw_target, 200, 4)
+	A.say("HYAH!")
+	return
+
+/datum/martial_art/plasma_fist/proc/Plasma(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)
+	A.do_attack_animation(D)
+	playsound(D.loc, 'sound/weapons/punch1.ogg', 50, 1, -1)
+	A.say("PLASMA FIST!")
+	D.visible_message("<span class='danger'>[A] has hit [D] with PLASMA FIST TECHNIQUE!</span>", \
+								"<span class='userdanger'>[A] has hit [D] with PLASMA FIST TECHNIQUE!</span>")
+	var/obj/item/organ/brain/B = D.getorgan(/obj/item/organ/brain)
+	if(B)
+		B.loc = get_turf(D)
+		B.transfer_identity(D)
+		D.internal_organs -= B
+	D.gib()
+	return
+
+/datum/martial_art/plasma_fist/harm_act(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)
+	add_element("H")
+	if(check_streak(A,D))
+		return 1
+	basic_hit(A,D)
+	return 1
+
+/datum/martial_art/plasma_fist/disarm_act(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)
+	add_element("D")
+	if(check_streak(A,D))
+		return 1
+	basic_hit(A,D)
+	return 1
+
+/datum/martial_art/plasma_fist/grab_act(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)
+	add_element("G")
+	if(check_streak(A,D))
+		return 1
+	basic_hit(A,D)
+	return 1
+
+
+//ITEMS
+
+/obj/item/clothing/gloves/boxing
+	var/datum/martial_art/old = null
+
+/obj/item/clothing/gloves/boxing/equipped(mob/user, slot)
+	if(!ishuman(user))
+		return
+	if(slot == slot_gloves)
+		var/mob/living/carbon/human/H = user
+		old = H.martial_art
+		H.martial_art = new/datum/martial_art/boxing(null)
+	return
+
+/obj/item/clothing/gloves/boxing/on_unequip(mob/user)
+	if(!ishuman(user))
+		return
+	var/mob/living/carbon/human/H = user
+	H.martial_art = old
+	return
+
+/obj/item/weapon/storage/belt/champion/wrestling
+	var/datum/martial_art/old = null
+
+/obj/item/weapon/storage/belt/champion/wrestling/equipped(mob/user, slot)
+	if(!ishuman(user))
+		return
+	if(slot == slot_belt)
+		var/mob/living/carbon/human/H = user
+		old = H.martial_art
+		H.martial_art = new/datum/martial_art/wrestling(null)
+	return
+
+/obj/item/weapon/storage/belt/champion/wrestling/on_unequip(mob/user)
+	if(!ishuman(user))
+		return
+	var/mob/living/carbon/human/H = user
+	H.martial_art = old
+	return
+
+/obj/item/weapon/plasma_fist_scroll
+	name = "Plasma Fist Scroll"
+	desc = "Teaches the traditional wizard martial art."
+	icon = 'icons/obj/wizard.dmi'
+	icon_state ="scroll2"
+	var/used = 0
+
+/obj/item/weapon/plasma_fist_scroll/attack_self(mob/user as mob)
+	if(!ishuman(user) || used)
+		return
+	var/mob/living/carbon/human/H = user
+	H.martial_art = new/datum/martial_art/plasma_fist
+	used = 1
+	desc += "It looks like it's magic was used up."
+
+/datum/martial_art/plasma_fist
+	name = "Plasma Fist"
+	var/streak = ""
```

Comment:
mind moving this (and any other "unique functionality" variables/procs) down to the base class?
So we can easily have other martial_arts do streaks, etc.